### ISSUE TITLE
Create an order - rows?

### ISSUE THREAD

**Issue by abutizi** [viktor2097/visma-administration#1]
Thanks a ton for this amazing library.

I would be eternally thankful if you could give an example of how to create a order/invoice row. I know how to create the actual order, but how do you assign rows to it?

Thanks in advance!

**Comment by viktor2097** [comment]
Unfortunately there's no super easy way to do that right now.
I'm in the progress of writing a solution and will update the package once i'm happy with the implementation.

You can however still create rows if you wanted by accessing the c# api with the .api attribute
```py
from visma_administration import Visma
from visma_administration import Api

with Visma.get_company_api("company") as api:
    # Create an invoice
    invoice = api.supplier_invoice_head.new()
    #... set some additional data on invoice

    # Create rows and assign them to invoice
    number_of_rows = 1
    rows = api.api.AdkCreateDataRow(api.api.ADK_DB_SUPPLIER_INVOICE_ROW, number_of_rows)
    api.api.AdkSetData(invoice.data, api.api.ADK_SUP_INV_HEAD_ROWS, rows)
    
    # Retrieve a row and assign some value to it
    row = api.api.AdkGetDataRow(rows, 0)
    err = api.api.AdkSetStr(row, api.api.ADK_OOI_ROW_ACCOUNT_NUMBER, String(account))

    invoice.adk_sup_inv_head_nrows = number_of_rows
    invoice.create()
```

Next update should add support for easily adding and removing rows.

**Comment by abutizi** [comment]
Thank you for your response and the provided example!
I look forward to the updated version. For now I will try the current approach.

**Comment by abutizi** [comment]
How about in case of getting invoice rows from an existing invoice?

**Comment by viktor2097** [comment]
> How about in case of getting invoice rows from an existing invoice?

This has been resolved in #2.

You can get rows from an existing invoice with the following code

```py
with Visma.get_company_api("company") as api:
    invoice = api.supplier_invoice_head.get(adk_sup_inv_head_invoice_number="85894")
    for row in invoice.rows():
        print(row.adk_ooi_row_account_number) # Row fields are accessible like any DB_FIELD instance now.   
```

Creating new rows for an invoice is equally easy.
```py
with Visma.get_company_api("company") as api:
    invoice = api.supplier_invoice_head.new()
    rows = invoice.create_rows(quantity=3)
    rows[0].adk_ooi_row_account_number = "5010"
    rows[1].adk_ooi_row_account_number = "5008"
    rows[2].adk_ooi_row_account_number = "4050"
    invoice.create()
```

There's some caveats to deleting rows, but check the readme
